### PR TITLE
[Intel MKL] Fixing a memory leak problem

### PR DESCRIPTION
**Pull request by mahmoud-abuzaina**
The changes in this PR fix a potential memory leak by reusing the same variable instead of creating new one each iteration. This change is owned by @guizili0 who can advise if it is possible to write a unit test for that.

### CODE REVIEW

**Comment by rthadur** [comment]
@mahmoud-abuzaina can you please include description of what changes are included , and also is it possible to write test cases around the changes ?

**Comment by mahmoud-abuzaina** [comment]
@rthadur I have updated the description.

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 1]
```diff
@@ -1646,6 +1657,9 @@ class MklQuantizedConv2DOp
   memory* input_bias_ = nullptr;
   memory* scaled_bias_ = nullptr;
 
+  Tensor scaled_bias_tensor;
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: Nit: class member naming. Same for `scaled_bias_buf_`.
```suggestion
  Tensor scaled_bias_tensor_;
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! I have a minor naming comment.

File: tensorflow/core/kernels/mkl_conv_ops.cc (hunk 1 above)
Comment (guizili0) [reply tensorflow/core/kernels/mkl_conv_ops.cc]: @penpornk Thanks for you comments, I have update my code based on it.

**Comment by mahmoud-abuzaina** [comment]
@googlebot I consent.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the quick fixes. One variable name hasn't been updated yet. 

@guizili0 Could you please also post `@googlebot I consent.`? Thank you very much!

**Comment by guizili0** [comment]
@googlebot I consent.

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 2]
```diff
@@ -1646,6 +1657,9 @@ class MklQuantizedConv2DOp
   memory* input_bias_ = nullptr;
   memory* scaled_bias_ = nullptr;
 
+  Tensor scaled_bias_tensor_;
+  void* scaled_bias_buf = nullptr;
```
[the remark this answers is not in the document]
Comment (guizili0) [reply tensorflow/core/kernels/mkl_conv_ops.cc]: @penpornk Updated, sorry about this.

**Review by penpornk — APPROVED** [review]
No worries, and thank you for your patience!

**Pull request merged**